pkgs/development/libraries/pico-sdk/default.nix

```
@@ -1,21 +1,21 @@
 { lib, stdenv, fetchFromGitHub, cmake }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "pico-sdk";
   version = "1.5.1";
 
   src = fetchFromGitHub {
     owner = "raspberrypi";
-    repo = pname;
-    rev = version;
-    sha256 = "sha256-JNcxd86XNNiPkvipVFR3X255boMmq+YcuJXUP4JwInU=";
+    repo = finalAttrs.pname;
```

```suggestion
    repo = "pico-sdk";
```

never use `finalAttrs.pname` in a url, see #326683